src/Domain/DonationRepository.php

```diff
@@ -72,7 +73,7 @@ public function doCreate(SalesforceWriteProxy $donation): bool
     public function doUpdate(SalesforceWriteProxy $donation): bool
     {
         if ($donation->getPaymentMethodType() === null) {
-            $donation->setPaymentMethodType('card');
+            $donation->setPaymentMethodType(PaymentMethodType::card->name);
```

Comment:
(copied from commit comments) 

I think it's generally better not to use the `name` of an enum for anything except debugging - it's sort of reflectiony. If you need to convert an enum to string make it a backed enum and use the `value` property. But as I've commented elsewhere I think this can just change to `$donation->setPaymentMethodType(PaymentMethodType::card);`